messenger/message-recorder.rst

```diff
@@ -1,7 +1,7 @@
 .. index::
     single: Messenger; Record messages; Transaction messages
 
-Transactional Messages: Handle Events After CommandHandler is Done
+Transactional Messages: Handle New Messages After Handling is Done
```

Comment:
It's a bit tricky: in Messenger, messages are called messages. If you're using Messenger as an event bus, you can of course call them events. I'm mostly using the word "message" to keep consistent with messenger (and not confuse with the EventDispatcher), though I keep "event" wording in the code examples.